src/molecules/formfields/ToggleField/index.js

```diff
@@ -35,13 +35,14 @@ function ToggleField( props ) {
       <div className={styles['button-wrapper']}>
         { toggleChoices.map( (choice, idx) =>
           <Button
-            variant='toggle'
+            variant={input.value === choice.value ? 'toggle-active' : 'toggle'}
             key={`button-toggle-btn-${idx}`}
             className={styles.button}
-            onClick={onClick}
-          >{ choice }</Button>
-          )
-        }
+            onClick={() => input.onChange(choice.value)}
```

Comment:
I'm confused by your PR comment; it seems to use this component w/o ReduxForm, one would still need to pass in a `input` prop in order for this `onClick` handler to work. Your comment states that one would only need to pass in `onChange`. Am I missing something or did I take your comment too literally?